### ISSUE TITLE
Support for end game

### ISSUE THREAD

**Issue by ForceGT** [kashish-jain/Tambola#59]
Hi can you add a button for ending the game?
The button should be able to force all the users out of the socket connections

**Comment by kashish-jain** [comment]
Should only the host has this functionality? Why do you think this feature is necessary? Have you come through such a use case?

**Comment by chirag1712** [comment]
I think it makes for the host to have this functionality instead of having to close all tabs and then starting a new game from scratch. Thoughts @ForceGT?

**Comment by ForceGT** [comment]
Yeah actually I meant if a host can have an option to end the game whenever needed

It will help the host to close the game whenever needed because for example if the host closes the tab then it should reflect in other players game as well

**Comment by chirag1712** [comment]
I think we have the functionality that you mentioned, "when a host closes the tab, it should reflect in other players' games as well". Correct me if I am wrong though @kashish-jain

**Comment by ForceGT** [comment]
I think you there is just a javascript alert , something like "Do you want to exit"


Instead of that, having an explicit button is always appreciable

**Comment by ForceGT** [comment]
Also, this is an awesome part time project
I would love to make an Android App for this

And would eagerly invite collaboration from you guys too

**Comment by kashish-jain** [comment]
Um, we already have a cross-platform app under progress. We can invite you for collaboration when it is in a good state.

**Comment by ForceGT** [comment]
That's going to be Javascript right?
React Native I'm guessing

**Comment by kashish-jain** [comment]
Yes
